pages/index.js

```diff
@@ -1,3 +1,36 @@
+import fetcher from "../helpers/fetcher";
+import useSWR from "swr";
+import styled from "styled-components";
+import Head from "next/head";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data, error } = useSWR("api/facilities", fetcher);
+
+  if (error) return <div>failed to load</div>;
+  if (!data) return <div>loading...</div>;
+
+  return (
+    <>
+      <Head>
+        <title>App-Name</title>
+      </Head>
+      <header></header>
```

Comment:
As a general rule of thumb, leave only that in the code which is immediately useful. An empty tag doesn't do anything for the app. Add it back in when it becomes useful :)